src/Analyzers/CSharp/CodeFixes/UseCollectionInitializer/CSharpUseCollectionInitializerCodeFixProvider.cs

```diff
@@ -36,80 +42,198 @@ public CSharpUseCollectionInitializerCodeFixProvider()
         }
 
         protected override StatementSyntax GetNewStatement(
+            SourceText sourceText,
             StatementSyntax statement,
             BaseObjectCreationExpressionSyntax objectCreation,
-            ImmutableArray<ExpressionStatementSyntax> matches)
+            int wrappingLength,
+            bool useCollectionExpression,
+            ImmutableArray<Match<StatementSyntax>> matches)
         {
             return statement.ReplaceNode(
                 objectCreation,
-                GetNewObjectCreation(objectCreation, matches));
+                GetNewObjectCreation(sourceText, objectCreation, wrappingLength, useCollectionExpression, matches));
```

Comment:
passed along as we prefer to not wrap collection expressions unless they get too long.